### DISCUSSION TITLE
Clarify Chapter.UseOffset meaning

### DISCUSSION THREAD

**Discussion by sandreas** [Zeugma440/atldotnet#181] [Q&A] [answered]
Hey there,

I know, that `ChapterInfo.UseOffset` does lead to a switch between `ChapterInfo.StartTime` and `ChapterInfo.StartOffset`, but I'm not sure how `ChapterInfo.StartOffset` must be handled.

Is `ChapterInfo.StartOffset` an absolute position value (offset from `0`) or relative to previous chapter? Example:

- 1st chapter has the following time based values: `startTime=0`, `endTime=5000`
- 2nd chapter is: `startTime=5000`, `endTime=10000`
- Given that the 2nd chapter is converted to useOffset=true, what what would be correct?:
  - `startOffset=0`, `endOffset=5000` (relative to the previous chapter)
  - `startOffset=5000`, `endOffset=10000` (absolute position)

**Comment by Zeugma440** [comment, marked as answer]
Hi there,

Offsets are specific to ID3v2 (see https://taglib.org/api/classTagLib_1_1ID3v2_1_1ChapterFrame.html#afbcafa2be34a8c7f740d5b3c3e1f6c20) and refer to an absolute byte offset.

They shouldn't be used for other tagging systems.
